src/components/Login/LocalLogin.tsx

```diff
@@ -13,6 +13,7 @@ const messages = defineMessages({
   loginerror: 'Something went wrong while trying to sign in.',
   signingin: 'Signing in…',
   signin: 'Sign in',
+  forgotpassword: 'Forgot password?',
```

Comment:
Title case for buttons! :)  "Forgot Password?"

The `signin` button string was also updated to "Sign In" but it didn't get changed here, if you want to fix that as well.

Reply:
It isn't a button now tho. It just has the same styling as a button. Is it the same for anchor links?

Reply:
Changed it to what you proposed now anyways. But I'd like to know.

Reply:
Sorry I was AFK!  But yes, it is the same for anchor links or other "actions" the user can perform :)